### ISSUE TITLE
Prefix the Spring Boot properties with something other than "spring."?

### ISSUE THREAD

**Issue by mjustin** [flamingock/mongock#254]
Given that Mongock isn't an official project by the Spring Framework, does it make sense for the Spring Boot application properties to be prefixed with `spring.`, e.g. `spring.mongock.change-logs-scan-package`?  Or would it make more sense for them to have their own top-level namespace, e.g. just `mongock.change-logs-scan-package`?

**Comment by dieppa** [comment]
This was designed that way because we had some early discussion with Spring team about integrating Mongock. So we decided to that to make it look more Spring friendly. But although we are still moving forward with that integration, I think you are right. When the integration is in place, that will added in their configuration.

Now it's the time to change that, later will be too late. Thanks

**Comment by mjustin** [comment]
Cool, regarding the conversations with the Spring team, though!

**Comment by dieppa** [comment]
done and documentation updated

**Comment by mjustin** [comment]
Looks like a couple spots were missed:

https://github.com/cloudyrock/mongock/blob/0dfdd76560aaed46114f52ef67d814bb223831cc/mongock-spring-v5/src/main/java/com/github/cloudyrock/spring/v5/MongockConfiguration.java#L81

```java
  public static class MongockLegacyMigrationConfig extends MongockLegacyMigration {

    @Override
    @ConfigurationProperties("spring.mongock.legacy-migration.mapping-fields")
    public LegacyMigrationMappingFields getMappingFields() {
      return super.getMappingFields();
    }
  }
```

https://github.com/cloudyrock/mongock/blob/0dfdd76560aaed46114f52ef67d814bb223831cc/mongock-spring-v5/src/main/java/com/github/cloudyrock/spring/v5/MongockContext.java#L13

```java
public class MongockContext {

  @Bean
  @ConditionalOnProperty(value = "spring.mongock.runner-type", matchIfMissing = true, havingValue = "ApplicationRunner")
  public MongockSpring5.MongockApplicationRunner mongockApplicationRunner(MongockSpring5.Builder mongockBuilder) {
    return mongockBuilder.buildApplicationRunner();
  }

  @Bean
  @ConditionalOnProperty(value = "spring.mongock.runner-type", havingValue = "InitializingBean")
  public MongockSpring5.MongockInitializingBeanRunner mongockInitializingBeanRunner(MongockSpring5.Builder mongockBuilder) {
    return mongockBuilder.buildInitializingBeanRunner();
  }
}
```

**Comment by dieppa** [comment]
it seems we did the integration tests agains the wrong artifact. My bad!

**Comment by dieppa** [comment]
solved in 4.1.11
